great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -3723,11 +3723,21 @@ def _construct_data_context_id(self) -> str:
 
     def _compile_evaluation_parameter_dependencies(self) -> None:
         self._evaluation_parameter_dependencies = {}
-        # NOTE: Chetan - 20211118: This iteration is reverting the behavior performed here:
-        # https://github.com/great-expectations/great_expectations/pull/3377
-        # This revision was necessary due to breaking changes but will need to be brought back in a future ticket.
+        # we have to iterate through all expectation suites because evaluation parameters
+        # can reference metric values from other suites
         for key in self.expectations_store.list_keys():
-            expectation_suite_dict: dict = cast(dict, self.expectations_store.get(key))
+            try:
+                expectation_suite_dict: dict = cast(
+                    dict, self.expectations_store.get(key)
+                )
+            except ValidationError as e:
+                # if a suite that isn't associated with the checkpoint compiling eval params is misconfigured
+                # we should ignore that instead of breaking all checkpoints in the entire context
+                logger.info(
```

Comment:
Shouldn't this be a warning?

Reply:
Yes it should. I changed this to a warning.